Scenarios.md

```diff
@@ -617,6 +657,11 @@
  * should not remove package from the lib directory
  * should still have the package file in the directory
 
+#### when uninstalling a package that forgets to call uninstall bin file
+
+ * should have had a shim
+ * should have removed the shim

```

Comment:
Awesome, thanks for doing this!

Reply:
Of course, this only applies to packages that install into `..\chocolatey\lib\<package-name>`, so shims targetting binaries somewhere else would not get removed.

If Shimgen accepted the package name as a parameter and stored this in the shim's FileVersionInfo data (together with the target binary, as suggested elsewhere), then this limitation disappears.